Dockerfile

```
@@ -62,6 +62,19 @@ RUN cd /ros2_ws/src/ros2_astra_camera/astra_camera/src/ && \
     's/calcAndPublishStaticTransform();/if (!publish_tf_) return; calcAndPublishStaticTransform();/g' \
     ob_camera_node.cpp
 
+# Create health check package
+RUN cd src/ && \
+    source /opt/ros/$ROS_DISTRO/setup.bash && \
+    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp std_msgs && \
```

Here should be lifecycle_msgs
```suggestion
    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp lifecycle_msgs && \
```